contracts/eth/nearprover/contracts/test/NearProofV2.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8;
+
+import "../NearProver.sol";
+
+contract NearProverV2 is NearProver {
+    function version() pure external returns (string memory) {
```

Comment:
The new testing contract should cover the following upgradeability cases:

- Changing the state variables values.
- Adding or overriding a functionality
- Expanding the storage layout (e.g defining new state variables).

The current testing upgradeability contract only covers one feature.